client/cmd.go

```diff
@@ -11,6 +11,23 @@ import (
 	"github.com/cosmos/cosmos-sdk/client/flags"
 )
 
+type contextKey string
```

Comment:
Why defining a string type alias?

Reply:
You need an addressable and comparable type for context keys.